From 849bd9830f760358e6276b10153d80e8fa766118 Mon Sep 17 00:00:00 2001
From: Arbeit Eins <eins@Niminal.(none)>
Date: Wed, 13 Jun 2012 15:49:22 -0700
Subject: [PATCH 1/1] Added USER_PATH to make patching pianobar easy.

---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d48a56e..008d774 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 # makefile of pianobar
 
-PREFIX:=/usr/local
+PREFIX:="${HOME}"/local
 BINDIR:=${PREFIX}/bin
 LIBDIR:=${PREFIX}/lib
 INCDIR:=${PREFIX}/include
-- 
1.7.9.5

