--- current_squid_config	2012-06-04 05:01:31.623118081 -0700
+++ /etc/squid3/squid.conf	2012-06-04 05:57:59.283742214 -0700
@@ -3691,6 +3691,7 @@
 #	seconds will receive a 'timeout' message.
 #Default:
 # shutdown_lifetime 30 seconds
+shutdown_lifetime 0 seconds
 
 # ADMINISTRATIVE PARAMETERS
 # -----------------------------------------------------------------------------
@@ -5795,3 +5796,43 @@
 #Default:
 # max_filedescriptors 0
 
+# A niemands production: force privoxy as squid's parent proxy
+cache_peer 127.0.0.1 parent 8118 7 no-query no-digest no-netdb-exchange
+
+# Make sure we're ALWAYS using Privoxy and Tor, and not bypassing the cache
+never_direct allow all
+
+# And let's remove some headers which can be used to divulge some of our personal info
+# Obsolete.
+#header_access From deny all
+#header_access Server deny all
+#header_access WWW-Authenticate deny all
+#header_access Link deny all
+
+# The following I got from http://www.christianschenk.org/blog/enhancing-your-privacy-using-squid-and-privoxy/
+# Obsolete.
+#header_access Cache-Control deny all
+#header_access Proxy-Connection deny all
+#header_access X-Cache deny all
+#header_access X-Cache-Lookup deny all
+#header_access Via deny all
+#header_access Forwarded-For deny all
+#header_access X-Forwarded-For deny all
+#header_access Pragma deny all
+#header_access Keep-Alive deny all
+
+# The following I got from http://forums.debian.net/viewtopic.php?f=16&t=59301
+request_header_access From deny all
+request_header_access Referer deny all
+request_header_access Server deny all
+request_header_access WWW-Authenticate deny all
+request_header_access Link deny all
+request_header_access Cache-Control deny all
+reply_header_access From deny all
+reply_header_access Referer deny all
+reply_header_access Server deny all
+reply_header_access User-Agent deny all
+reply_header_access WWW-Authenticate deny all
+reply_header_access Link deny all
+via off
+forwarded_for delete
